### ISSUE TITLE
Following your guide I get the following error.

### ISSUE THREAD

**Issue by larsrbak** [shopgun/shopgun-android-sdk#4]
Caused by: java.lang.ClassNotFoundException: Didn't find class "com.shopgun.android.utils.PackageUtils"

**Comment by oizo** [comment]
If you have added the `shopgun-android-sdk` via jCenter, like so:
```groovy
dependencies {
    compile 'com.shopgun.android:sdk:3.2.2'
}
```
Gradle should automatically resolve all dependencies that the sdk has, in this case to our [markhor project](https://github.com/shopgun/markhor/). If the dependencies aren't automatically resolved, please try [refreshing all gradle projects](https://www.jetbrains.com/help/idea/2016.2/synchronizing-changes-in-gradle-project-and-intellij-idea-project.html) and then 'clean' your project via "Build -> Clean project".

**Comment by jmintb** [comment]
I am experiencing the same issue, any other ideas to get around it?

**Comment by oizo** [comment]
The library dependencies are unfortunately not correctly included in the release to maven, due to incorrect setup in bintray.gradle. I'm notthe dev anymore, and doesn't have the finger on the release button anymore, so for the v3.2.2 release the workaround would to either include all dependencies from the lib manually:
```
    compile 'com.shopgun.android:utils:1.0.0'
    compile 'com.android.support:support-v4:23.3.0'
    compile 'de.greenrobot:eventbus:2.4.0'
    compile 'com.squareup.okhttp:okhttp:2.7.5'
    compile 'com.squareup.picasso:picasso:2.5.2'
```
or clone the repo and import the sdk as a module.

**Comment by oizo** [comment]
Unfortunately the gradle script for uploading to maven doesn't include the library dependencies as part of the pom file - hence the missing class. For now you'll need to clone the repo, and include the module the old fashioned way. I'm no longer the maintainer, and cannot upload a fix. @AgneseBussone will need to look into this.

**Comment by oizo** [comment]
Should be resolved in this #5

**Comment by AgneseBussone** [comment]
The issue should be resolved with the 4.0.1-beta.

```
dependencies {
       implementation 'com.shopgun.android:sdk:4.0.1-beta'
}
```

In case you already tried this version, please force gradle to download it again with
`./gradlew build --refresh-dependencies`
